tests/buildings_bulk_load_comparisons_complex.pg

```diff
@@ -0,0 +1,167 @@
+           (1062, 2040), (1062, 2041),
+           (1067, 2067), (1068, 2067),
+           (1079, 2083), (1080, 2083)$$,
+  'Check return of buildings_bulk_load find related function'
+);
+
+--5----------------------------------------------------------
+-- Test comparisons function inputs complex added outlines
+
+SELECT set_has(
+    'SELECT bulk_load_outline_id FROM buildings_bulk_load.added',
+    $$VALUES (2038), (2051), (2052), (2084)$$,
+    'Check added outlines after comparisons function has run'
+);
+
+--6----------------------------------------------------------
+-- Test comparisons function inputs complex removed outlines
+
+SELECT set_has(
+    'SELECT building_outline_id FROM buildings_bulk_load.removed',
+    $$VALUES (1042), (1058), (1081)$$,
+    'Check removed outlines after comparisons function has run'
+);
+
+--7----------------------------------------------------------
+-- Test comparisons function inputs complex matched outlines
+-- by building_outline_id
+SELECT set_has(
+  'SELECT building_outline_id FROM buildings_bulk_load.matched',
+  $$VALUES (1061), (1060),
+           (1041), (1048),
+           (1056)$$,
+  'Check building_outline_id of matched table after comparisons function has run'
+);
+
+--8----------------------------------------------------------
+-- Test comparsisons function inputs complex related outlines
+-- by bulk_load_outline id
+
+SELECT set_has(
+  'SELECT bulk_load_outline_id FROM buildings_bulk_load.matched',
+  $$VALUES (2053), (2050),
+           (2047), (2045),
+           (2042)$$,
+  'Check bulk_load_outline_ids of matched table after comparisons function has run'
+);
+
+--9----------------------------------------------------------
+-- Test comparisons function inputs complex related outlines
+-- by building_outline_id 
+
+SELECT set_has(
+  'SELECT building_outline_id FROM buildings_bulk_load.related',
+  $$VALUES (1059), (1057),
+           (1059), (1059),
+           (1059), (1038),
+           (1040), (1039),
+           (1043), (1045),
+           (1053), (1054),
+           (1055), (1044),
+           (1046), (1047),
+           (1049), (1050),
+           (1051), (1052),
+           (1062), (1062),
+           (1062), (1062)$$,
+  'Check building_outline_id of related table after comparisons function has run'
+);
+
+--10----------------------------------------------------------
+-- Test comparisons function inputs complex related outlines
+-- by bulk_load_outline_id
+
+SELECT set_has(
+  'SELECT bulk_load_outline_id FROM buildings_bulk_load.related',
+  $$VALUES (2056), (2055),
+           (2055), (2054),
+           (2049), (2048),
+           (2048), (2046),
+           (2046), (2046),
+           (2044), (2044),
+           (2044), (2043),
+           (2043), (2043),
+           (2043), (2043),
+           (2043), (2043),
+           (2041), (2040),
+           (2039), (2037)$$,
+  'Check bulk_load_outline_id of related table after comparisons function has run'
+);
```

Comment:
Add a newline at the end of the file so that git doesn't get confused in future.